lambdas/service/vendor/static/swagger-ui.html.template.mustache

```diff
@@ -43,11 +43,14 @@
         plugins: [
           SwaggerUIBundle.plugins.DownloadUrl
         ],
-        oauth2RedirectUrl: "{{OAUTH2_REDIRECT_URL}}"
```

Comment:
I noticed the transition from `{{` (HTML-escaped) to `{{{` (literal). If that's a bug fix it should be committed separately. If not, I would like to know what the motivation actually is for the transition.

Reply:
I changed line 48 because we switched from pasting the contents of the string inside of quotes that were already present in the HTML to pasting the entire literal, including the surrounding quotes, to accommodate the option to provide literals that are not strings, i.e. `null`. The quotes are HTML-escaped when using `{{}}`, resulting in rendered HTML that looks like `client_id = &quot;4732`.

The change to line 46 wasn't necessary as there are no escaped characters.

Reply:
I think `{{` was the wrong choice even before this change, though. We can not be sure that a URL does not contain any characters that need to be escaped. The same goes for the ID. We should have treated both as opaque strings and escaped them using the proper mechanism (which in the old code would NOT have been HTML-escaping but escaping double quotes) OR embed the ID as JSON without the surrounding quotes.

Reply:
Technically, we would also need to handle all kinds of edge cases caused by the fact that this is JS embedded in HTML. However, I think those scenarios are very unlikely considering that WE control the value of these variables. If the values originated from outside the system, say, a user request, we couldn't take that stance. In that case, we wouldn't attempt to handle the edge cases (that would be incredibly [complicated](https://uploadcare.com/blog/vulnerability-in-html-design/)), we would simply not interpolate into JS.